engineio/src/error.rs

```diff
@@ -30,20 +30,20 @@ pub enum Error {
     InvalidUrlScheme(String),
     #[error("Error during connection via http: {0}")]
     IncompleteResponseFromReqwest(#[from] ReqwestError),
+    #[error("Error with websocket connection: {0}")]
+    AsyncWebsocketError(#[from] TungsteniteError),
```

Comment:
nit: is the fact that this error came from an async context relevant to users pattern matching on this error?

Reply:
Good point!